hbase-server/src/main/java/org/apache/hadoop/hbase/util/FSUtils.java

```diff
@@ -546,19 +544,19 @@ public static void setVersion(FileSystem fs, Path rootdir, String version,
    * @throws IOException if checking the FileSystem fails
    */
   public static boolean checkClusterIdExists(FileSystem fs, Path rootdir,
-      int wait) throws IOException {
+      long wait) throws IOException {
     while (true) {
       try {
         Path filePath = new Path(rootdir, HConstants.CLUSTER_ID_FILE_NAME);
         return fs.exists(filePath);
       } catch (IOException ioe) {
-        if (wait > 0) {
-          LOG.warn("Unable to check cluster ID file in " + rootdir.toString() +
-              ", retrying in "+wait+"msec: "+StringUtils.stringifyException(ioe));
+        if (wait > 0L) {
+          LOG.warn("Unable to check cluster ID file in {}, retrying in {}ms", ioe);
```

Comment:
this is missing the params for `rootdir` and `wait`.

Reply:
Thanks!